tools/autograd/gen_annotated_fn_args.py

```diff
@@ -77,17 +77,34 @@ def gen_annotated(
 
 @with_native_function
 def gen_annotated_args(f: NativeFunction) -> str:
+    def _get_kwargs_func_exclusion_list():
+        # functions that don't work with kwargs in test_overrides.py
+        return [
+            "diagonal",
+            "round_",
+            "round",
+            "scatter_",
+        ]
```

Comment:
End of day I'm not able to get all ops works with `kwargs` hence creating this exclusion list to move forward. The goal of this PR is to unblock shipping #97417